Core/src/org/sleuthkit/autopsy/communications/relationships/ContactNode.java

```diff
@@ -136,14 +138,20 @@ protected Sheet createSheet() {
 
         return sheet;
     }
-    
+
     private void addPropertiesToSheet(String propertyID, Sheet.Set sheetSet, List<BlackboardAttribute> attributeList) {
         int count = 0;
         for (BlackboardAttribute bba : attributeList) {
             if (count++ > 0) {
                 sheetSet.put(new NodeProperty<>(propertyID + "_" + count, bba.getAttributeType().getDisplayName(), "", bba.getDisplayString()));
+                if (bba.getAttributeType().getTypeName().startsWith("TSK_PHONE")) {
+                  sheetSet.put(new NodeProperty<>(Bundle.ContactNode_Country() + "_" + count, bba.getAttributeType().getDisplayName() + " " + Bundle.ContactNode_Country(), "", PhoneNumUtil.getCountryCode(bba.getValueString())));  
```

Comment:
The country could be added to the end of the phone number attribute.  I put a comment in the Jira with a screenshot.

Reply:
Fixed it and pushed it.